addons/test_mail/tests/test_mail_channel.py

```diff
@@ -191,6 +191,35 @@ def test_channel_classic_recipients(self):
             self.test_channel.message_post(body="Test", message_type='comment', subtype_xmlid='mail.mt_comment')
         self.assertSentEmail(self.test_channel.env.user.partner_id, [self.test_partner])
 
+    def test_channel_seen(self):
```

Comment:
add docstring comment for the test similar to: "In case of concurrent `channel_seen` RPC, ensure the oldest call has no effect."